### PR TITLE
[gh-#632] hotfix for auth account find

### PR DESCRIPTION
**Pull request by khaliqgant**
cc/ @bastienbeurier

### CODE REVIEW

File: packages/server/lib/controllers/oauth.controller.ts [hunk 1]
```diff
@@ -446,36 +446,13 @@ class OAuthController {
...
-                activity_log_id: errorActivityLog as number,
-                content: WSErrBuilder.MissingConnectionId().message,
-                timestamp: Date.now(),
-                params: {
-                    ...errorManager.getExpressRequestContext(req)
-                }
-            });
+            logger.log('error', errorMessage);
 
             errorManager.report(e, { metadata: errorManager.getExpressRequestContext(req) });
```
Comment (bastienbeurier) [inline packages/server/lib/controllers/oauth.controller.ts:454]: I think `errorManager.report` already logs the error, so I don't think we need the previous `logger.log` call.

**Comment by bastienbeurier** [comment]
It might also make sense to bump the patch version of the `packages/server/package.json` to `0.17.1` since we already had a `0.17.0` staging Docker version and I just pushed `0.17.1` with the hotfix.

File: packages/server/lib/controllers/oauth.controller.ts (hunk 1 above)
Comment (khaliqgant) [reply packages/server/lib/controllers/oauth.controller.ts:454]: Yup, resolved with https://github.com/NangoHQ/nango/pull/634/commits/f661510cad820e7a2b86fddeb35793326290315c

**Comment by khaliqgant** [comment]
> It might also make sense to bump the patch version of the `packages/server/package.json` to `0.17.1` since we already had a `0.17.0` staging Docker version and I just pushed `0.17.1` with the hotfix.

Done with https://github.com/NangoHQ/nango/pull/634/commits/3c2705871f2a19370db278eb49b5ce5ec953fcfd

**Review by bastienbeurier — APPROVED** [review]

**Pull request merged**